services/core-web/src/components/noticeOfWork/applications/permitGeneration/NOWPermitGeneration.js

```diff
@@ -509,16 +581,27 @@ export class NOWPermitGeneration extends Component {
         >
           <>
             {isProcessed ? (
-              <LoadingWrapper condition={this.state.isLoaded}>
+              <LoadingWrapper condition={this.state.isLoaded || this.props.isLoaded}>
```

Comment:
We don't want these conditions to be when both the state and props isLoaded are true?

Reply:
ah yes, good point, should be &&.